librubyfmt/src/format.rs

```diff
@@ -678,7 +678,12 @@ pub fn format_list_like_thing_items(
             //raise "this is bad" if expr[0] == :tstring_content
```

Comment:
we should probably delete this line lol

Reply:
I can delete that whole comment block if you don't think its relevant context anymore.

Reply:
Done in #284